suite-common/wallet-utils/src/cardanoUtils.ts

```diff
@@ -1,5 +1,23 @@
+import BigNumber from 'bignumber.js';
+
+import {
+    Account,
+    Output,
+    PrecomposedTransactionFinal,
+    PrecomposedTransactionFinalCardano,
+    PoolsResponse,
+    StakePool,
+} from '@suite-common/wallet-types';
+import { CARDANO, CardanoCertificate, CardanoOutput, PROTO } from '@trezor/connect';
 import { Network } from '@suite-common/wallet-config';
 
+import {
+    amountToSatoshi,
+    formatAmount,
+    formatNetworkAmount,
+    networkAmountToSatoshi,
+} from './accountUtils';
+
 // NOTE: This was moved from packages/suite/src/utils/wallet/cardanoUtils.ts
 // Reason for that is that we need to use it in @suite-common/ package, but we cannot import
 // @fivebinaries/coin-selection there, because it will break mobile app bundling.
```

Comment:
This comment is redundant now, would you please remove it?

Reply:
Sure 337bbad74c